### PR TITLE
Very simple test set for training

### PR DESCRIPTION
**Pull request by Ttl**
I saw some discussion about the test set for training. I've been missing it too so here's a simple implementation.

parse.py now takes optional "--test" argument for path to chunk to be used as a test set. Symmetry is still applied to it and it loops around during the testing so the tested positions are not identical. I'm not expert in tensorflow and didn't find a quick method to solve those, but this should still be better than nothing.

### CODE REVIEW

**Comment by gcp** [comment]
See #747. It's similar but I split the chunk list and use a TF conditional to select the right feeder.

**Comment by Ttl** [comment]
Looks better than this.

Does the splitting work correctly when training from supervised data? When chunking from supervised data leelaz loops over the dataset several times so positions from games get split over several chunks. When the split is done positions from same games might end in both sets.

**Comment by gcp** [comment]
FWIW I don't think shuffling the test set is needed, you don't care about the correlation during testing.

**Comment by gcp** [comment]
>Does the splitting work correctly when training from supervised data?

No.

Given the presence of shuffle buffers in TensorFlow, the supervised learning dumper should probably be changed to chunk entire games.

**Comment by Ttl** [comment]
I really don't want to chunk everything again since it takes so long. I would want to have an optional command line argument for chunks to use as a test set instead of splitting. It doesn't seem to be very difficult change.

You can merge your pull request and I'll add the option later. I'll close this one.

**Comment by gcp** [comment]
>It doesn't seem to be very difficult change.

No, and it would work nicely with the code in the other pull. It just needs the chunks split somehow.

**Comment by gcp** [comment]
>FWIW I don't think shuffling the test set is needed, you don't care about the correlation during testing.

Uhm, on the other hand this probably does mess up the MSE measurement. So it probably also needs shuffling.
